test/simulation/README.md

```diff
@@ -8,38 +8,63 @@ The simulation tests’ only dependencies are [Docker](https://docs.docker.com/i
 
 ## Usage (via `xud` npm scripts)
 
-Run the tests after building all images:
+#### Run the tests after building all images:
 
 ```bash
 $ npm run test:sim:build
 $ npm run test:sim:run
 ```
 
-Whenever test code is being refactored, one must build the `test` image before re-running the tests so that changes could take effect:
+#### Specific builds
+
+For taking into effect changes in test code, one must build the `test` image before re-running the tests:
 
 ```bash
 $ npm run test:sim:build:test
 $ npm run test:sim:run
 ```
---- 
 
-The `test:sim:run` script mentioned above runs only the Integration tests suite.
+For taking into effect changes in `xud` (non-test, production code), one must build the `xud` image before re-running the tests, in addition to cleaning its volume:
+
+```bash
+$ docker volume rm simulation_xud-vol
+$ npm run test:sim:build:test
```

Comment:
Isn't this line supposed to be `npm run test:sim:build`? Or it this where `npm run test:sim:build:xud` would be appropriate?

Reply:
> Or it this where npm run test:sim:build:xud would be appropriate?

Yes, my bad.